packages/blade/src/components/Alert/Alert.tsx

```diff
@@ -0,0 +1,217 @@
+import type { ReactChild, ReactElement } from 'react';
+import { useState } from 'react';
+
+import StyledAlert from './StyledAlert';
+import { AlertIcon, CloseIcon } from '~components/Icons';
+import { makeAccessible } from '~utils';
+import { IconButton } from '~components/Button/IconButton';
+import Box from '~components/Box';
+import { Heading, Text } from '~components/Typography';
+import BaseButton from '~components/Button/BaseButton';
+import { BaseLink } from '~components/Link/BaseLink';
+
+type Nullable<Type> = Type | null;
+
+type PrimaryAction = {
+  text: string;
+  onClick: () => void;
+};
+
+type SecondaryActionButton = {
+  text: string;
+  onClick: () => void;
+};
+
+type SecondaryActionLinkButton = {
+  text: string;
+  href: string;
+  onClick?: () => void;
+  target?: string;
+  /**
+   * When `target` is set to `_blank` this is automatically set to `noopener noreferrer`
+   */
+  rel?: string;
+};
+
+type SecondaryAction = SecondaryActionButton | SecondaryActionLinkButton;
+
+export type AlertProps = {
+  /**
+   * Body content, pass text or JSX. Avoid passing components except `Link` to customize the content.
+   */
+  description: ReactChild;
+
+  /**
+   * A brief heading
+   */
+  title?: string;
+
+  /**
+   * Shows a dismiss button
+   *
+   * @default true
+   */
+  isDismissable?: boolean;
+
+  /**
+   * A callback when the dismiss button is clicked
+   */
+  onDismiss?: () => void;
+
+  /**
+   * Can be set to `high` for a more prominent look. Not to be confused with a11y contrast.
+   *
+   * @default low
+   */
+  contrast?: 'high' | 'low';
+
+  /**
+   * Makes the Alert span the entire container width, instead of the default max width of `584px`
+   *
+   * @default false
+   */
+  isFullWidth?: boolean;
+
+  /**
+   * Sets the color tone
+   *
+   * @default information
+   */
+  intent?: 'positive' | 'negative' | 'information' | 'notice';
```

Comment:
We defined the `IntentTypes` but not using them.

Reply:
These are being used for various colors, please check the rendering block.

Reply:
Changed to using `Feedback` in https://github.com/razorpay/blade/pull/666